pkg/kubelet/oom_watcher.go

```diff
@@ -49,10 +50,11 @@ const systemOOMEvent = "SystemOOM"
 func (ow *realOOMWatcher) Start(ref *api.ObjectReference) error {
 	request := events.Request{
 		EventType: map[cadvisorapi.EventType]bool{
-			cadvisorapi.EventOom: true,
+			cadvisorapi.EventOom:     true,
+			cadvisorapi.EventOomKill: true,
 		},
 		ContainerName:        "/",
-		IncludeSubcontainers: false,
+		IncludeSubcontainers: true,
```

Comment:
@vishh @euank @yujuhong @feiskyer 

In our deployment `cgroup` file system is mounted like this
```
cgroup on /sys/fs/cgroup/net_cls,net_prio type cgroup (rw,nosuid,nodev,noexec,relatime,net_cls,net_prio)
cgroup on /sys/fs/cgroup/perf_event type cgroup (rw,nosuid,nodev,noexec,relatime,perf_event)
cgroup on /sys/fs/cgroup/cpu,cpuacct type cgroup (rw,nosuid,nodev,noexec,relatime,cpu,cpuacct)
cgroup on /sys/fs/cgroup/blkio type cgroup (rw,nosuid,nodev,noexec,relatime,blkio)
cgroup on /sys/fs/cgroup/freezer type cgroup (rw,nosuid,nodev,noexec,relatime,freezer)
cgroup on /sys/fs/cgroup/devices type cgroup (rw,nosuid,nodev,noexec,relatime,devices)
cgroup on /sys/fs/cgroup/cpuset type cgroup (rw,nosuid,nodev,noexec,relatime,cpuset)
cgroup on /sys/fs/cgroup/memory type cgroup (rw,nosuid,nodev,noexec,relatime,memory)
```

Under `/sys/fs/cgroup/memory` there are three sub directories 
* docker
* system.slice
* user.slice

I am not familiar with cAdvisor, but according to the webui it provides, iiuc, it uses these sub directories name as a prefix when naming or splitting cgroup statistics.

Maybe our deployment is not the corresponding one for kubelet to query OOM events from cAdvisor correctly. This may be related to the cgroup hierarchy. 

Can you guys give some advices?